src/core/server/core_app/core_app.ts

```diff
@@ -49,6 +51,35 @@ export class CoreApp {
       });
     });
 
+    router.get(
+      {
+        path: '/{path*}',
+        validate: {
+          params: schema.object({
+            path: schema.maybe(schema.string()),
+          }),
+          query: schema.maybe(schema.recordOf(schema.string(), schema.any())),
+        },
+      },
```

Comment:
This was the last route remaining in legacy. I'm still honestly unsure of its value, but we're preserving the exact same behavior.

Reply:
Is it a trailing slash removal added by @spalger in https://github.com/elastic/kibana/commit/615f4c972af37dd7731e8d58302a38210cd806cc ?
I'd add a comment about the purpose of it